onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -883,6 +894,16 @@ def _where_input_wrangler(
         dtypes=[torch.float16],
         reason="fixme: ONNX Runtime aborted",
     ),
+    TorchLibOpInfo(
+        "nn.functional.embedding_bag",
+        core_ops.aten_embedding_bag,
+        input_wrangler=_embedding_bag_input_wrangler,
+        trace_only=True,
+        tolerance={torch.float16: (1e-3, 1e-2)},
+    ).skip(
+        matcher=lambda sample: "padding_idx" in sample.kwargs or "max_norm" in sample.kwargs,
+        reason="padding_idx is used for training mode, and max_norm is rare case.",
```

Comment:
Looks like max_norm is not part of the aten signature. I assume we can safely ignore them?

```suggestion
        reason="padding_idx or max_norm is not part of the aten signature.",
```

Reply:
https://github.com/pytorch/pytorch/pull/105862 shows very likely `max_norm` triggers an extra `aten.embedding_renorm.default` operator. 

Based on latest discussion, we want to exclude only cases when `padding_idx` is None or `max_norm` is not None.

Reply:
Do you mean padding_idx is not None?

Reply:
No, I thought after discussion the torchlib op signature expects `padding_idx` as int with default, so we only skip tests when `padding_idx` is explicitly None (this case may not exist)?

Reply:
yes, `padding_idx` could be wrangled around. Semantically it doesn't affect forward so yea all cases should work.

Reply:
- When ```padding_idx``` is None, the logic just be what I have implemented.
- When ```padding_idx``` has value, (such as  -1, 0, 1, or 2), it will affect the final result. So the ```-1``` cannot be default value.

Reply:
The analysis makes sense to me. I'm not certain. My impression on `-1` is only that it is the default value for `padding_idx` in `native_functions.yaml`. It depends on torch implementation regarding `padding_idx`. 

If we are not implementing `padding_idx` case, do we raise error when `padding_idx` is provided?

Reply:
We tend to not raise in torchlib. This is a trace only function so it may be more relaxed, but I wouldn't rely on it. 

Let me check the -1 case.

Reply:
OK so it looks like -1 doesn't have a special meaning (from https://github.com/pytorch/pytorch/blob/487ebcac3bc10b4b4b0631dafe2a12ddb0852f2a/torch/csrc/api/src/nn/modules/embedding.cpp#L105). In this case I suggest we do nothing to support `-1`, and add a `ValueError` for the time being, with a `TODO` next to it.

Reply:
So far, from my try:
```
when padding_idx is not None(e.g. padding_idx=2 in test case 6):
    when in eager mode: actual == expected
    when in full graph mode: actual != expected
```
So it might be an ORT bug.